src/shared/components/styles/index.tsx

```diff
@@ -0,0 +1 @@
+import "./index.scss";
```

Comment:
Let's move the `styles` outside of the components directory. 
shared
   - components
   - styles

And in a follow up we should decide on a few aliases for all this stuff

Reply:
I dig that, was following M$ lead here but it def felt weird since there are no components here.